lib/ProductOpener/Import.pm

```diff
@@ -1172,12 +1176,13 @@ sub import_csv_file ($args_ref) {
 
 							if (exists_taxonomy_tag($field, $tagid)) {
 								my $opposite_tagid = get_property($field, $tagid, "opposite:en");
+								# we have an opposite, add it
 								if (defined $opposite_tagid) {
 									$tag_to_add = $opposite_tagid;
 								}
 							}
 						}
-
+						# add the value to eventual existing values
```

Comment:
``
```suggestion
						# If we have a tag to add (because we had a "yes" or a "no" value for a specific tag field), concatenate it to possible pre-existing tags
```

Reply:
éventuel -> possible, not eventual